narwhal/types/src/tests/batch_serde.rs

```diff
@@ -58,7 +54,7 @@ fn test_serde_batch() {
 }
 
 #[test]
-fn test_bincode_serde_batch() {
+fn test_bcs_serde_batch() {
```

Comment:
I am not so sure that those tests add more value anymore. Especially since we remove the (unused/obsolete) method `serialized_batch_digest` I think it we'll be ok to remove those tests here.

Reply:
Thanks will do!